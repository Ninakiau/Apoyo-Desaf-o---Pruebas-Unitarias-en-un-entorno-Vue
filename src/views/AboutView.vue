<template>
    <div>
        <h1>Este es el about</h1>
    </div>
</template>