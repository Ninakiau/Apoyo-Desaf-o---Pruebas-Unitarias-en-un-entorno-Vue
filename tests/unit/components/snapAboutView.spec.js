import { shallowMount } from "@vue/test-utils";
import AboutView from "@/views/AboutView.vue";

//Validación de match con el snapshot
describe('Vista Home', () => {
    //Se valida la similitud con el snapshot
    test('Validación de match con el snapshot', () => {
        const wrapper = shallowMount(AboutView)
        expect(wrapper.html()).toMatchSnapshot()
    })
})
