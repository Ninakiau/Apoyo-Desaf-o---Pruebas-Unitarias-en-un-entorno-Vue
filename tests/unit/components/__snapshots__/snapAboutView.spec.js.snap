// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vista Home Validaci√≥n de match con el snapshot 1`] = `
<div>
  <h1>Este es el about</h1>
</div>
`;
